client/asset/dcr/dcr_test.go

```diff
@@ -2439,7 +2439,10 @@ func TestSyncStatus(t *testing.T) {
 }
 
 func TestPreSwap(t *testing.T) {
-	wallet, node, shutdown, _ := tNewWallet()
+	wallet, node, shutdown, err := tNewWallet()
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
There's no need to do this everywhere. `tNewWallet` really shouldn't have an error return value. I can't remember why it was even added (instead of a `panic`, for instance), but we're not going to check it everywhere. Did you run into an instance where ignoring the error was troublesome?

Reply:
Could get rid of the error return, but it was wrong to have the defer before the error check

Reply:
> Did you run into an instance where ignoring the error was troublesome?

Nah, just being consistent with other part of the tests.

> Could get rid of the error return, 

I agree.